src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -252,21 +251,23 @@ private static bool TryGetGrpcMethod(Activity activity, out string grpcMethod)
 
         private static void AddGrpcAttributes(Activity activity, string grpcMethod, HttpContext context)
         {
-            // TODO: Should the leading slash be trimmed? Spec seems to suggest no leading slash: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md#span-name
-            // Client instrumentation is trimming the leading slash. Whatever we decide here, should we apply the same to the client side?
-            // activity.DisplayName = grpcMethod?.Trim('/');
```

Comment:
I'd forgotten to resolve this when I originally did the work on this. The spec seems to indicate that the name not lead with a trailing `/`, but the ASP.NET Core instrumentation sets the name appropriately already, though has a leading `/`. My take is that we just leave it this way. Unless others feel the spec clearly indicates that this `/` should not be in the name.